src/index.ts

```diff
@@ -52,3 +73,6 @@ export { PromiseMonitor } from './lib/promise/PromiseMonitor';
 export { HttpPromiseMonitor } from './lib/promise/HttpPromiseMonitor';
 // synchronized http promise
 export { SynchronizedHttpPromise } from './lib/promise/SynchronizedHttpPromise';
+
+// to remove in future major release
```

Comment:
Il faudrait mettre un TODO et dire plutôt : TODO to remove in the next major release, the 3.x.x

Reply:
C'est fait